spec/forms/nsm/steps/letters_calls_form_spec.rb

```diff
@@ -71,14 +71,14 @@
         let(:calls) { '' }
 
         it 'have an error' do
-          expect(subject).to be_valid
+          expect(subject).not_to be_valid
```

Comment:
These tests no longer test what they're supposed to be testing (although I will grant that they're so badly written it's hard to tell what they're supposed to be testing). Rather than just change the expectations here, I think we should change the setup so that the expectations remain valid _and_ add extra tests that demonstrate that in a different context we get a different result.

Reply:
The way that test description was wrote it expects to have an error but was actually validating true on it _not_ having an error 😄 so i guess this fix was just aligning that. Ok, no worries i'll have a look at extra tests